app/src/main/java/edu/wisc/ece/pinpoint/pages/map/MapFragment.java

```diff
@@ -221,4 +242,32 @@ private void getDeviceLocation() {
             });
         }
     }
+
+    private void setPinnieCount() {
+        FirebaseDriver driver = FirebaseDriver.getInstance();
+        String uid = driver.getCurrentUser().getUid();
```

Comment:
I don't think we should need this UID if we remove the UID argument from the pinnie fetching functions

Reply:
Oh gotcha I misunderstood what you were saying about the UID last night - I'll rework that this afternoon